R/utils.R

```diff
@@ -734,20 +734,47 @@ read_shims <- function() {
 
 active_shims <- function() {
 
-  objects <- purrr::map(ls(name = .GlobalEnv), get)
+  text <- "get_active_shims_fertile <- function(){
 
-  shims <- objects[purrr::map(objects, attr, "package") == "fertile"]
+  env_obj <- ls(.GlobalEnv)
 
-  func_attributes <- purrr::map(shims, attributes)
 
+  shims <- list()
+  shims_names <- c()
+  for (i in 1:length(env_obj)){
```

Comment:
Is it really necessary to do this with a `for` loop? Can't we do this by mapping a function over the list?

Reply:
I had tried that in my previous iteration and it wasn't working, but that might have been due to the environment problem rather than the mapping. Will try to convert to map to see if it works.

Reply:
Just updated this. See the screenshot of the new function in response to your other comment!